dataframe/dataframe.go

```diff
@@ -27,6 +27,46 @@ type Field struct {
 // Fields is a slice of Field pointers.
 type Fields []*Field
 
+// AppendRow adds a new row to the Frame by appending to each element of vals to
+// the corresponding Field in the dataframe.
+// The dataframe's Fields and the Fields' Vectors must be initalized or AppendRow will panic.
+// The number of arguments must match the number of Fields in the Frame and each type must coorespond
+// to the Field type or AppendRow will panic.
+func (f *Frame) AppendRow(vals ...interface{}) {
```

Comment:
Do we want to keep this?
Why someone would prefer this over the `AppendRowSafe`?

Reply:
Speed. It should be faster to use if you already know that both your data and your frame will meet what is checked for, bypassing all the checks.